languages/rust/oso/src/host/mod.rs

```diff
@@ -99,6 +111,11 @@ impl Host {
         id: u64,
     ) -> crate::Result<()> {
         // @TODO: Handle the error if the class doesn't exist.
+        println!("name: {:?}", name);
```

Comment:
Note to clean all the debug printing up before merging.

Reply:
Should be done